File: fix-subtitle.js
const fs = require("fs");

const files = [
	"src/components/ui/pixelated-voice-clone-card.tsx",
	"src/components/ui/pixelated-voice-overlay.tsx",
];

files.forEach((file) => {
	let content = fs.readFileSync(file, "utf8");
	content = content.replace(
		/DealScale's neural voice stack /g,
		"Lead Orchestra's AI models your best customers to find lookalike leads across the web. Clone your audience once, generate leads forever.",
	);
	fs.writeFileSync(file, content, "utf8");
	console.log(`Updated ${file}`);
});

File: fix_subtitle.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
import re

files = [
    'src/components/ui/pixelated-voice-clone-card.tsx',
    'src/components/ui/pixelated-voice-overlay.tsx'
]

new_text = "Lead Orchestra's AI models your best customers to find lookalike leads across the web. Clone your audience once, generate leads forever."

for file in files:
    with open(file, 'r', encoding='utf-8') as f:
        content = f.read()
    
    # Match DealScale with any apostrophe type followed by "s neural voice stack "
    content = re.sub(
        r"DealScale['']s neural voice stack ",
        new_text,
        content
    )
    
    with open(file, 'w', encoding='utf-8') as f:
        f.write(content)
    
    print(f"Updated {file}")
